index.html

```diff
@@ -1,42 +1,56 @@
-I will not proofread this.
-
-Some other things about me:
-
-I'm currently proudly and intensely hyperfixated on Pokemon Go and Wizard 101
-
-My favorite show is Supernatural
-
-My favorite movie is Ginger Snaps
-
-connect:
-
-github - vsspnkkvr
-email - vsspnkkvr@proton.me
-or - npetroli@unca.edu
-
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>vsspnkkvr</title>
+</head>
+<body>
+
+    <h1>About me:</h1>
+
+    <p>The first thing you should know about me is that I hate writing about myself; 
+    I get mad at the screen in front of me and my writing usually turns into ranting. 
+    I apologize in advance for briefly throwing you into the trenches of my inner war. 
+    The second thing about me is that I'm an autistic trans girl and therefore innately 
+    destined to be a programmer. God, I wish that was true. I do sincerely love programming 
+    and I'm getting better at it every day, but what I really am at heart is a writer. Not 
+    that I'm innately good at writing either — if I ever believed that, I'd face eternal 
+    writer's block — no, I'm a writer in that I see stories everywhere: in songs, in people, 
+    behind every forgettable object, every menial tasks; I see stories in code. The path of 
+    logic the computer travels, it's like a narrative to me, a hero's journey, and writing 
+    that code is just as much of a story too. When you're finished with a project, you have 
+    what all stories are: an expression. An expression of self, of intention, of action. 
+    This is a perspective that's hard to describe; it doesn't make much sense to me either, 
+    but what makes more sense is what drives it. I've felt excluded and outcasted from the 
+    world for much of my life and so I have a deep need to feel like my presence and 
+    existence impact it in some tangible way. The most fulfilling way I've found to achieve
+    that is by making something. Taking something from within myself and seeing it manifest
+    outside of myself. Making it real, and therefore, making myself real. If the fabric 
+    of the universe is woven out of stories, adding more stories is the way to become 
+    part of it. And that's why I'm trying to become what you would call a programmer. 
+    I'm an autistic writer who thinks the language of computers is just as beautiful if not
+    more so than the language of books. Thank you for your patience and for indulging me. 
+    I will not proofread this.</p>
+
+    <p>Some other things about me:</p>
+
+    <ul>
+        <li>I'm currently proudly and intensely hyperfixated on Pokemon Go and Wizard 101</li>
+
+        <li>My favorite show is Supernatural</li>
+
+        <li>My favorite movie is Ginger Snaps</li>
+    </ul>
+    
+    <p>connect:</P>
```

Comment:
This also seems like a heading so also probably want to wrap this in a heading element like `<h2>`